### DISCUSSION TITLE
Got Approved --> Still No Package Received

### DISCUSSION THREAD

**Discussion by muranovicharis** [community/community#136839] [GitHub Education]
### Hi everyone,

I was recently approved for the GitHub Education benefits (which I’m super excited about!), but I haven't received my GitHub Education package yet. I’m just wondering how long it typically takes for the package to arrive after getting approved.

If anyone else has gone through this process, could you share how long it took for your package to arrive? Should I be expecting an email or some other notification about it?

Thanks in advance for your help!

Best,
Haris Muranovic

**Comment by agustrodmar** [comment]
Hi @muranovicharis 

First of all, congrats on getting approved for the GitHub Education benefits!

Once approved, you should have immediate access to these benefits. You can check if you have access by visiting the [GitHub Education page] https://education.github.com/pack while logged into your account. 

If you’re not seeing them, it might be worth reaching out to GitHub support or revisiting the approval confirmation you received to ensure everything is OK.

Hope that helps

**Comment by davevad93** [comment]
Hi @muranovicharis ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @muranovicharis was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by muranovicharis** [reply]
<img width="323" alt="Bildschirmfoto 2024-08-26 um 15 35 24" src="https://github.com/user-attachments/assets/14f48460-05d9-49ab-8bb2-d7da7ea82ad9">
i have this message already 3 days

**Reply by muranovicharis** [reply]
it is saying approved but i still dont have the pack

**Reply by muranovicharis** [reply]
ik that it is saying "soon" but in the form it says it should take longer then 3 days

**Reply by agustrodmar** [reply]
Yeah! It looks like it. As it's said here #55559 the process is a little bit broken, sometimes it takes even 6 days without updates.

**Reply by davevad93** [reply]
That's right @muranovicharis , your status says your benefits will be "`available soon`" which means you still have to wait, when the benefits become available, the status label that says "`Approved`" will turn purple and state "`your benefits are now available`".

![0000](https://github.com/user-attachments/assets/4d3bdbe4-7f03-4567-abbf-4548fb4c76b0)

The waiting time may vary, most people get their benefits within 1 week but in some cases it can take longer, the discussions I have provided above are examples of people who have experienced the same as you and in the end received their benefits in less than 2 weeks. 
Don't worry. 👍
